src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search/umb-mini-search.html

```diff
@@ -1,9 +1,10 @@
 <ng-form class="umb-mini-search" ng-class="{'--has-value': vm.model !== null && vm.model !== ''}" novalidate>
-    <i class="icon icon-search" aria-hidden="true"></i>
+    <label for="mini-search"><i class="icon icon-search"></i><span class="sr-only"><localize key="general_typeToSearch">Search...</localize></span></label>
```

Comment:
Any reason you've removed `aria-hidden="true"` on the icon? These icons can not be properly articulated by screen readers and should be hidden. Especially now that you're adding a nice label around them! 🙂